src/interactive.ts

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ *
+ */
+
+import * as yargs from 'yargs'
+import * as inquirer from 'inquirer'
+import * as fs from 'fs'
+import {ActionsOnGoogle, AssistResponse} from './actions-on-google'
+
+class Interactive {
+
+    async main(): Promise<void> {
+        const argv = yargs
+            .usage('Usage: action-interactive [Options] [Action] [Prompt]')
+            .example('action-interactive -c ./credentials.json "my test app"', '')
+            .option('credential', {
+                alias: 'c',
+                description: 'Your credential file path',
+                type: 'string',
+            })
+            .option('locale', {
+                alias: 'l',
+                description: 'Locale string',
+                type: 'string',
+            })
+            .version(false)
+            .argv
+        const actionName = argv._.length >= 1 ? argv._[0] : undefined
+        const prompt = argv._.length >= 2 ? argv._[1] : undefined
+        const credential = argv.credential || './credentials.json'
+        const locale = argv.locale || 'en-US'
```

Comment:
Can we also add to description that the default locale is 'en-US'?

Reply:
Sure. I intend to add that. Thank you for the advice.